.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -20,4 +20,30 @@
 
 ### Other / What else might be useful to know?
 - _Provide any additional context or information about the changes made and their implementation._
-- _BE GENEROUS to your reviewer and your future self!_
\ No newline at end of file
+- _BE GENEROUS to your reviewer and your future self!_
+
+### Checks to be considered:
+- [ ] Performed a *self-review* of the code
+- [ ] Additional *automated test* coverage
+  - [ ] Unit tests
```

Comment:
I think  the main context needs to be checked, and the details underneath should be presented as bullet points IMO?